README.md

```diff
@@ -1,98 +1,170 @@
 
-- You may need more packages depending on your current environment setup
+#### Starting Minikube
 
-### Starting minikube
-From a terminal with administrator access run the following command:
->$minikube start
+Run the following command in your terminal:
+
+``` bash
+minikube start
+```
 
 If minikube fails to start, use the minikube documentation to troubleshoot any issues 
 https://minikube.sigs.k8s.io/docs/start/
 
-### Setting up ArgoCD regcred
- change directory to the regcred-init directory and run the following command:
+#### ArgoCD
+
+ArgoCD is a continuous deployment (CD) tool used to manage and provide automatic updates for Kubernetes applications
+
+The opal team uses Argo CD to ensure OPAL deployments are always up-to-date
+
 
->$helm install argoregcred . --create-namespace -n argocd
+##### Setting up ArgoCD regcred
+
+change directory to the regcred-init directory and run the following command:
+
+``` bash
+helm install argo-regcred . --create-namespace -n argocd
+```
+
+##### Install ArgoCD helm chart
 
-### Install ArgoCD helm chart
 now that the argoCD values are configured properly for a local deployment, helm install the argocd from the parent directory of ArgoCD repository
 
->$helm install argo ./chart -n argocd
+``` bash
+cd argo/chart
+helm install argo . -n argocd
+```
+
+
+##### Access the ArgoCD secret
 
-### Access the ArgoCD secret
 To access the password for the argocd login you will need to run the following command:
-> $kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
 
-the output from this command will be used on the argoCD dashboard interface
+``` bash
+kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
+```
+
+Alternatively, using `K9s`:
+1. Press `0` to ensure all namespaces are visible
+2. Type `:secret` and press enter
+4. Move your cursor down to `argocd-initial-admin-secret` and press `x`
+5. Copy the password for the next step
+
+##### Accessing the ArgoCD Dashboard
 
-### Add Repo to ArgoCD Dashboard
 In order to get argoCD dashboard working before applications are added, you need to start a port-forward service with kubectl:
->$kubectl port-forward service/argo-argocd-server -n argocd 8080:443
 
-Access the argoCD interface by opening a browser and navigate to the following:
+``` bash
+kubectl port-forward service/argo-argocd-server -n argocd 8080:443
+```
+
+Alternatively, using `K9s`:
+1. Press `0` to ensure all namespaces are visible
+2. Type `:svc` and press enter
+4. Move your cursor down to `argocd-argocd-server` and press `shift-f`
+5. Ensure `Container Port` says `server::8080`. If not, delete the line and set it to `8080`
+6. Move your cursor down to `OK` and press enter
+
+
+Access the argoCD interface by opening a browser and navigating to the following:
+
 > localhost:8080
 
-there should be a login page for argoCD
+This should take you to the login page for argoCD
+
 >username: **admin**
```

Comment:
This formatting is inconsistent with similar text elsewhere in the document. Preferred might look like

Username:
> admin

Password:
> output from the last section (remove the trailing '%' symbol if present)